hy/importer.py

```diff
@@ -306,17 +304,6 @@ def load_module(self, fullname=None):
             mod.__loader__ = self
             return mod
 
-        def _reopen(self):
-            """Same as `pkgutil.ImpLoader`, with an extra check for Hy
-            source"""
-            super(HyLoader, self)._reopen()
-
-            # Add the Hy case...
-            if self.file and self.file.closed:
-                mod_type = self.etc[2]
-                if mod_type == HY_SOURCE:
-                    self.file = io.open(self.filename, 'rU', encoding='utf-8')
```

Comment:
Not so sure about removing this override; the default `ImpLoader._reopen` simply does `open(self.filename, 'rU')`, which won't necessarily use the unicode codec (I think).

Reply:
Hmm, yeah, that could lead to trouble. You could try writing a test with a source file that has non-ASCII characters in it.

Reply:
Or is this already implicitly tested by all the other native tests that have non-ASCII characters?

Reply:
It is, but most likely not in a situation where `ImpLoader._reopen` is called.